src/lib/bootstrap.js

```diff
@@ -41,8 +40,4 @@ if (browserSupport) {
   s.type = 'module';
   s.src = '/' + entrypoint;
   document.head.append(s);
-} else {
-  // If we've transitioned into becoming an unsupported browser, then any
```

Comment:
We still need this, for the reason listed in the comment.

Reply:
Are there browsers that support service workers but not modules? I figure older versions of the evergreen browsers did but how many versions back are they?

Reply:
no, but I think this comment and code path should still be included since as we up the requirement level (e.g. Shadow DOM), it's obvious that old browser should transition away from having a SW.